### ISSUE TITLE
Can we also hide logger commands (python)?

### ISSUE THREAD

**Issue by jslomkowski** [estruyf/vscode-hide-comments#15]
It would be useful to also hide logger commands. What do you think?

Example before:
```
logger.info("deploying AciWebservice..")
aci_config = AciWebservice.deploy_configuration(cpu_cores=1, memory_gb=1)
logger.info("AciWebservice deployed!")

logger.info("setting InferenceConfig..")
env = Environment('custom')
```
Example after:
```

aci_config = AciWebservice.deploy_configuration(cpu_cores=1, memory_gb=1)



env = Environment('custom')
```

**Comment by estruyf** [comment]
Could be a setting that these lines could be hidden. Won't do it automatically, but other languages might need something similar.

**Comment by amagid** [comment]
I'm looking for an extension to do this right now. I'm writing primarily TypeScript with a configurable logger system. I keep my logger calls in my code so that if I am debugging code in production, I can enable debug and get lots of log data about the failure.

Could this extension be set up to accept a regexp parameter for identifying the lines to be hidden? In some of my code, the lines look like:
```ts
this._debugLogger.warn('my message')
```
But in other places it might be under a different name like:
```ts
logger.error('my error')
```

If I could configure this extension to say something like:
```json
"lineIdentificationRexexp": "(_debugLogger)|(logger)\.\w+\("
```
that would catch all of my logger calls throughout the code and hide them.

**Comment by estruyf** [comment]
Released in version 1.6.0
